### PR TITLE
minor enhancements to roadmap

### PR DESCRIPTION
**Pull request by animeshsingh**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/230)

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -41,6 +42,10 @@ Objective: "Improve the Serverless Experience by reducing cold starts/stops to 1
 * Server Shutdown
     * Ensure that all model servers shutdown within 10 seconds of not receiving traffic.
 
+Objective: "Simplify User Experience."
```
Comment (ellis-bigelow) [inline ROADMAP.md]: Let's make this objective more crisp. 

Simplify user experience with handling credentials for storage backends.

**Comment by ellis-bigelow** [comment]
Mostly LGTM

File: ROADMAP.md [hunk 2]
```diff
@@ -50,6 +55,10 @@ Objective: "Improve the Serverless Experience by reducing cold starts/stops to 1
     * Model Experimentation.
     * Ensembling.
     * Multi Arm Bandit.
+* Payload Logging
+    * Finalize the design and implementation for [Payload Logging](https://docs.google.com/document/d/1MBl5frM9l_wyQkYEaDeHOP6Mrsuz9YOob7276AAN9_c/edit?usp=sharing)
+    * Leverage existing infrastructure (e.g. Istio Sidecars) as much as possible
```
Comment (ellis-bigelow) [inline ROADMAP.md]: Let's remove this design detail.

File: ROADMAP.md [hunk 3]
```diff
@@ -50,6 +55,10 @@ Objective: "Improve the Serverless Experience by reducing cold starts/stops to 1
...
+    * Provide ability to support multiple storage mechanisms
```
Comment (ellis-bigelow) [inline ROADMAP.md]: And this.

File: ROADMAP.md [hunk 4]
```diff
@@ -41,6 +42,10 @@ Objective: "Improve the Serverless Experience by reducing cold starts/stops to 1
 * Server Shutdown
     * Ensure that all model servers shutdown within 10 seconds of not receiving traffic.
 
+Objective: "Simplify User Experience."
+* Secure storage mechanisms
+    * Explore simplifying user experience with storage backends protected by credentials (e.g. S3/GCS accounts with credentials)
```
Comment (ellis-bigelow) [inline ROADMAP.md]: Is there something more actionable than "explore". Are we designing something? What specifically are we aiming to improve. Perhaps just "Address the storage pains discussed in #XYZ"

Comment (animeshsingh) [reply ROADMAP.md]: Changed it to implement, and can be discussed how.

File: ROADMAP.md (hunk 3 above)
Comment (animeshsingh) [reply ROADMAP.md]: done

File: ROADMAP.md (hunk 2 above)
Comment (animeshsingh) [reply ROADMAP.md]: done

File: ROADMAP.md (hunk 1 above)
Comment (animeshsingh) [reply ROADMAP.md]: done

**Comment by ellis-bigelow** [comment]
/approve
:/

**Pull request merged**